scp/base_melinder.py

```diff
@@ -36,13 +36,10 @@ class BaseMelinder(BaseFluid):
     )
 
     @abstractmethod
-    def calc_freeze_point(self, x: float) -> float:
+    def coefficient_freezing(self) -> Tuple:
```

Comment:
So all Melinder fluids must override this to return the coefficients for their particular fluid.  :+1: